src/components/calculator.js

```diff
@@ -8,59 +8,54 @@ import calculate from '../logic/calculate';
  ,jsx-a11y/no-static-element-interactions
  ,react/prop-types
  ,react/no-unused-state */
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-    this.performClick = this.performClick.bind(this);
-  }
+let [obj, setObj] = [{}, () => {}];
```

Comment:
- [ ] Kindly remove this declaration as it serves no purpose in your application. You already have a state managing the obj inside your calculator component